kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/spec/style/scopes/ContainerScope.kt

```diff
@@ -98,6 +98,23 @@ interface ContainerScope : TestScope {
       afterAny(f)
    }
 
+   /**
+    * Registers a callback that is executed once for this container after all tests have completed.
+    *
+    * This differs from [afterContainer] which is executed after each container including child containers.
+    */
+   fun after(f: suspend (TestCase) -> Unit) {
```

Comment:
should we rename `after` to `afterAllTestsInContainerCompleted`? I'm sure there is a better name than what I've suggested, but IMO just `after` is not clear enough

Reply:
The name was the longest part of the discussion in #2792 
I'm open to anything, there's `afterScope` perhaps. I don't think we want something too wordy, like `afterContaineratThisLevelOnly`

Reply:
`afterScope` sounds good

Reply:
I'm always interested in improving naming. In this case, I don't feel like I'm able to meaningfully contribute unless I have a complete visualization of our test hierarchy along with corresponding lifecycle methods (listeners, interceptors, whatever). Would be cool if we could get that documented completely in one place.

Reply:
afterScope it is
And will make a PR to  document all our callbacks.